app/assets/javascripts/filters.js

```diff
@@ -10,7 +10,8 @@ $(document).ready(function () {
   });
 
   $('#filter_unlimited').change(function () {
-    $('#search').prop('disabled', $(this).prop('checked'));
+    const isChecked = $(this).prop('checked');
```

Comment:
I'm assuming const will fail here (as this code is not managed by webpack, would it make sense to move it under webpack control?

Reply:
This break` the Rails asset precomling: `Uglifier::Error: Unexpected token: keyword (const). To use ES6 syntax, harmony mode must be enabled with Uglifier.new(:harmony => true).`

I'd suggest to use ES5 only here.

Reply:
@mmoll ok, good to know :)

Reply:
or move it to webpack control as suggested before :)